java/README.md

```diff
@@ -0,0 +1,26 @@
+#Word Count in Java
+===================
+Get Vagrant up and running from parent dir:
+	
+	$ ./bootstrap_vagrant.sh
+  	$ vagrant ssh
+  	...
+  	vagrant@vagrant-ubuntu-trusty-64:~/$ cd /vagrant/java/
+  
+Manual SBT install inside Vagrant:
+
+  	$ echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
```

Comment:
Maybe we can add this to the ansible yml? 

https://github.com/concord/getting-started/blob/master/ansible/dev.yml#L21

That way the user doesn't have to do anything w/ while inside the vagrant. Just ./bootstrap_vagrant.sh and it should be done for him. 

Let me know.